concent_api/core/validation.py

```diff
@@ -434,3 +430,33 @@ def validate_frames(frames_list: List[int]):
                 'Frame number must be grater than 0',
                 ErrorCode.MESSAGE_FRAME_VALUE_NOT_POSITIVE_INTEGER
             )
+
+
+def validate_compute_task_def(compute_task_def: message.tasks.ComputeTaskDef) -> None:
+    string_fields = ["output_format", "scene_file"]
+    integer_fields = ["frames"]
```

Comment:
I think this variable name is misleading in comparison to previous one. In `string_fields` you have list of fields which values should be strings, so it suggest that with `integer_fields` you have a list of fields which values should be integers. But it should be a list of integers. So maybe call it `list_of_integer_fields` or simply `other_mandatory_fields` because later you don't check a type of values from those fields.

Reply:
good catch; done